tests/server/auth/test_auth.py

```diff
@@ -56,6 +59,49 @@ def test_authenticate(client, monkeypatch):
         client.search_experiments()
 
 
+def _mlflow_search_experiments_rest(base_uri, headers):
+    response = requests.post(
+        f"{base_uri}/api/2.0/mlflow/experiments/search",
+        headers=headers,
+        json={
+            "max_results": 100,
+        },
+    )
+    response.raise_for_status()
+    return response
+
+
+@pytest.mark.parametrize(
+    "client",
+    [
+        {
+            "MLFLOW_AUTH_CONFIG_PATH": "tests/server/auth/fixtures/jwt_auth.ini",
+            "PYTHONPATH": str(Path(__file__).parent.parent.parent.parent / "examples" / "jwt_auth"),
```

Comment:
```suggestion
            "PYTHONPATH": str(Path.cwd() / "examples" / "jwt_auth"),
```

This is simpler (but assumes we run test in the repository root and fails if you don't).